src/test/java/org/testeditor/fixture/commons/text/generate/NameGeneratorTest.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.commons.text.generate;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+class NameGeneratorTest {
+
+    @Test
+    void generateRandomFirstNameTest() {
```

Comment:
The methods that do not use a seed are inherently hard to test. What do you do if, after the tenthousand's time, this test goes red? Its not reproducible… 

Maybe you could:
1. add a message to the second assert that includes the value of `randomName`.
2. in the production code, always log the seed that is being used.